README.md

```diff
@@ -29,6 +29,8 @@ var generatedQuery = generator.stringify(parsedQuery);
 ```
 Set `sparqlStar` to `true` to allow [SPARQL*](https://blog.liu.se/olafhartig/2019/01/10/position-statement-rdf-star-and-sparql-star/) syntax.
 
+Set `pathOnly` to `true` to parse SPARQL paths such as `foaf:name/foaf:knows` rather than the full SPARQL Algebra.
```

Comment:
Note that the `pathOnly` option does not enforce only paths; so perhaps better `acceptPaths` or so?

Reply:
Or of course the reverse, where it MUST be a path.

Reply:
> Note that the pathOnly option does not enforce only paths.

That's exactly what the second error is for - I just don't have good test coverage for that part yet

Reply:
Makes sense!